cognite/client/data_classes/agents/agent_tools.py

```diff
@@ -240,6 +240,15 @@ class SummarizeDocumentAgentToolUpsert(AgentToolUpsert):
 
     _type: ClassVar[str] = "summarizeDocument"
 
+    @classmethod
+    def _load(
+        cls, resource: dict[str, Any], cognite_client: CogniteClient | None = None
+    ) -> SummarizeDocumentAgentToolUpsert:
+        return cls(
+            name=resource["name"],
+            description=resource["description"],
+        )
+
```

Comment:
Note to reviewer: These changes are somewhat unrelated, but weren't caught in the PR that added the data classes.

Adding it in this PR because tests that catch this issue only run when the data classes are in use